### PR TITLE
[TAS-2525] 💄 Purchase CTA button improvement 

### PR DESCRIPTION
**Pull request by AuroraHuang22**
- [x] - edition button
- [x] - cta button 
- [x] - column left UI
- [x] - gift dialog
- [x] - discount + stock label

截圖 2024-11-13 下午3 03 42

截圖 2024-11-13 下午3 48 09

### CODE REVIEW

File: src/components/NFTEditionSelectItemV2.vue [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        <div class="text-like-green text-[16px] font-500">{{ name }}</div>
+      </div>
+    </div>
+
+    <div class="flex flex-co whitespace-nowrap">
+      <div v-if="discountInfo" class="text-like-green text-[12px]">
+        {{ discountInfo.originalPriceLabel }}
+      </div>
+      <div class="text-like-green text-[16px] font-500">
+        {{ priceLabel }}
```
Comment (nwingt) [inline src/components/NFTEditionSelectItemV2.vue:40]: Please use `v-text` for content

File: src/components/NFTEditionSelectItemV2.vue [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        />
+        <IconCircleCheck
+          v-if="isSelected"
+          class="text-like-green w-[20px] absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2"
+        />
+      </div>
+      <div class="flex flex-col">
+        <div>
+          <NFTStockLabel :stock="stock" />
+        </div>
```
Comment (nwingt) [inline src/components/NFTEditionSelectItemV2.vue:30]: What is the extra div for?

File: src/components/NFTEditionSelectItemV2.vue [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        <div class="text-like-green text-[16px] font-500">{{ name }}</div>
+      </div>
+    </div>
+
+    <div class="flex flex-co whitespace-nowrap">
```
Comment (nwingt) [inline src/components/NFTEditionSelectItemV2.vue:35]: ???
```suggestion
    <div class="flex flex-col whitespace-nowrap">
```

File: src/components/NFTEditionSelect.vue [hunk 4]
```diff
@@ -1,128 +1,58 @@
...
+        :text="$t('nft_edition_select_confirm_button_text_add_to_cart')"
+        @click="handleClickAddToCartButton"
       >
-        <template #prepend>
-          <IconGift class="w-[16px]" />
-        </template>
+        <IconAdd class="w-[16px]" />
+        <p class="ml-[8px]">
+          {{ $t('nft_edition_select_confirm_button_text_add_to_cart') }}
+        </p>
```
Comment (nwingt) [inline src/components/NFTEditionSelect.vue:35]: use `v-text` for content

File: src/components/NFTEditionSelect.vue [hunk 5]
```diff
@@ -1,128 +1,58 @@
...
-          :stock="stock"
-          :is-dark="!isAllSoldOut"
-        />
-      </template>
-
+      class="hidden laptop:block border-b-[1px] border-[#ebebebeb] w-full my-[16px]"
+    />
+    <div class="text-[18px] font-500 text-dark-gray w-full">
+      {{ $t('nft_edition_select_section_label') }}
+    </div>
```
Comment (nwingt) [inline src/components/NFTEditionSelect.vue:8]: use `v-text` for content

File: src/components/NFTEditionSelect.vue [hunk 6]
```diff
@@ -1,128 +1,58 @@
...
-          priceLabel
-        }}</span>
-        <NFTStockLabel
-          v-if="items.length > 1 || isAllSoldOut"
-          :stock="stock"
-          :is-dark="!isAllSoldOut"
-        />
-      </template>
-
+      class="hidden laptop:block border-b-[1px] border-[#ebebebeb] w-full my-[16px]"
```
Comment (nwingt) [inline src/components/NFTEditionSelect.vue:4]: is border with transparency?

File: src/components/NFTEditionSelectItemV2.vue (hunk 1 above)
Comment (AuroraHuang22) [reply src/components/NFTEditionSelectItemV2.vue]: [💄 Add discount original price label](https://github.com/likecoin/liker-land/pull/1948/commits/e8fdc2a56a0301fbd7904ec5cdb9f9cdd33c73a7)

File: src/components/NFTEditionSelect.vue (hunk 4 above)
Comment (AuroraHuang22) [reply src/components/NFTEditionSelect.vue]: [💄 Add discount original price label](https://github.com/likecoin/liker-land/pull/1948/commits/e8fdc2a56a0301fbd7904ec5cdb9f9cdd33c73a7)

File: src/components/NFTEditionSelect.vue (hunk 5 above)
Comment (AuroraHuang22) [reply src/components/NFTEditionSelect.vue]: [💄 Add discount original price label](https://github.com/likecoin/liker-land/pull/1948/commits/e8fdc2a56a0301fbd7904ec5cdb9f9cdd33c73a7)

File: src/components/NFTEditionSelect.vue (hunk 6 above)
Comment (AuroraHuang22) [reply src/components/NFTEditionSelect.vue:4]: seems not?
截圖 2024-11-13 下午2 19 19

File: src/components/NFTEditionSelectItemV2.vue (hunk 2 above)
Comment (AuroraHuang22) [reply src/components/NFTEditionSelectItemV2.vue]: [💄 Add discount & stock label](https://github.com/likecoin/liker-land/pull/1948/commits/0b283a00f60115d455b5c26431cd2e414980c814)

File: src/components/NFTEditionSelectItemV2.vue (hunk 3 above)
Comment (AuroraHuang22) [reply src/components/NFTEditionSelectItemV2.vue]: [💄 Add discount & stock label](https://github.com/likecoin/liker-land/pull/1948/commits/0b283a00f60115d455b5c26431cd2e414980c814)

File: src/components/NFTEditionSelect.vue (hunk 6 above)
Comment (nwingt) [reply src/components/NFTEditionSelect.vue:4]: Why is the hex value `#ebebebeb`?

Comment (AuroraHuang22) [reply src/components/NFTEditionSelect.vue]: [💄Correct divider color](https://github.com/likecoin/liker-land/pull/1948/commits/3ee58a2b4cc38fd7608914dc9f9c51279b51cce0)

**Review by williamchong — CHANGES_REQUESTED** [review]

**Review by williamchong — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

**Pull request merged**